lib/logstash/java_integration.rb

```diff
@@ -39,3 +39,9 @@ def self.===(other)
     super
   end
 end
+
+module java::util::Collection
+  def delete(o)
+    self.removeAll([o]) ? o : block_given? ? yield : nil
```

Comment:
why the conversion to single element array? can't we just call `self.remove(o)` ?